src/createAdder.js

```diff
@@ -20,7 +20,17 @@
  * @param {number} initialValue
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  let previous = 0;
+
+  return (...args) => {
+    const current = args.length > 0
+      ? args.reduce((a, b) => a + b) + previous
+      : 0 + previous;
```

Comment:
```suggestion
    const current = args.reduce((a, b) => a + b, previous);
```
я думаю это будет работать также